SettingsList.py

```diff
@@ -3571,16 +3571,27 @@ def __init__(self, name, gui_text, min, max, default, step=1,
                          ''',
         shared         = True,
     ),
-    Checkbutton(
-        name           = 'correct_chest_sizes',
-        gui_text       = 'Chest Size Matches Contents',
-        gui_tooltip    = '''\
-            Chests will be large if they contain a major
-            item and small if they don't. Boss keys will
-            be in gold chests. This allows skipping
-            chests if they are small. However, skipping
-            small chests will mean having low health,
-            ammo, and rupees, so doing so is a risk.
+    Combobox(
+        name           = 'correct_chest_appearances',
+        gui_text       = 'Chest Appearance Matches Contents',
+        default        = 'off',
+        choices        = {
+            'off': 'Off',
+            'textures': 'Textures',
+            'sizes': 'Size and Textures'
+        },
+        gui_tooltip    = '''\
+            Chest style will reflect its contents regardless
+            of size.  Fancy chests will contain keys, Gilded
+            chests will contain major items, shuffled tokens
+            will be in Webbed chests, and Wooden chests
+            will contain the rest.
+            This allows skipping chests if they are wooden. 
+            However, skipping wooden chests will mean having 
+            low health, ammo, and rupees, so doing so is a risk.
+            Size matches content will change chests with major
```

Comment:
Use the option name in quotes. 'Size and Textures' will change...

Reply:
Added quotes and made consistent with option name change.